feature_engine/base_transformers.py

```diff
@@ -144,3 +147,14 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
     # for the check_estimator tests
     def _more_tags(self):
         return _return_tags()
+
+    def get_feature_names(self) -> List:
+        """
+        Return feature names for output features.
+
+        -------
```

Comment:
We need to use the doscstring convention that I use in all files. We can not have ---- without a title. Here it should be Parameters i think. Pls check other files.